package.json

```diff
@@ -53,7 +53,7 @@
     "react-dom": "^18.2.0",
     "react-idle-timer": "^5.7.2",
     "react-router-dom": "^6.9.0",
-    "soul-cli": "^0.6.1",
+    "soul-cli": "0.7.3",
```

Comment:
is it worth including the up-arrow, so we get soul updates automatically?

Reply:
Okay, I will add this.